repository_cli/cli/records.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 Graz University of Technology.
+#
+# repository-cli is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Management commands for records."""
+
+import json
+
+import click
+from flask.cli import with_appcontext
+from flask_principal import Identity
+from invenio_rdm_records.proxies import current_rdm_records
+
+from .click_options import option_identifier, option_pid
+from .util import get_identity
+
+
+@click.group()
+def records():
+    """Management commands for records."""
+    pass
+
+
+@click.group()
+def identifiers():
+    """Management commands for record identifiers."""
+    pass
+
+
+records.add_command(identifiers)
+
+
+@identifiers.command("list")
+@option_pid
+@with_appcontext
+def list_identifiers(pid):
+    """List record's identifiers."""
+    identity = get_identity()
+    service = current_rdm_records.records_service
+    record_data = service.read(id_=pid, identity=identity).data.copy()
+    current_identifiers = record_data["metadata"].get("identifiers", [])
+    for index, identifier in enumerate(current_identifiers):
+        # BLACK, RED, GREEN, YELLOW, BLUE, MAGENTA, CYAN, WHITE, RESET
+        fg = "yellow" if index % 2 == 0 else "cyan"
+        click.secho(json.dumps(identifier, indent=2), fg=fg)
```

Comment:
*request:* add output response for empty identifiers, for example, if the record does not have a field of identifiers.